scripts/Diffraction/isis_powder/osiris_routines/osiris_algs.py

```diff
@@ -0,0 +1,300 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import isis_powder.routines.common as common
+from isis_powder.routines.run_details import create_run_details_object, get_cal_mapping_dict
+import numpy as np
+from mantid.simpleapi import NormaliseByCurrent, CropWorkspace, RebinToWorkspace, Minus, Divide, DiffractionFocussing, \
+    ConvertUnits, MergeRuns, GroupWorkspaces
+from mantid.api import AnalysisDataService
+
+
+d_range_with_time = {'drange1': [11700, 51700],
```

Comment:
is it ok for these to be hard coded? Will we ever add more?

Reply:
These are taken from the OSIRIS user manual. to my knowlage there o plans on changing them